scripts/prepare-test-data.sh

```diff
@@ -37,3 +37,18 @@ else
     cd ..
     echo "   ... finished preparing dataset 'data/example-visium-v2'."
 fi
+
+
+# Download and extract Visium v2 dataset.
+if [ -d example-visium-v1 ]; then
```

Comment:
we're repeating `example-visium-v1` quite a bit

on #3730 i put
```
name="soma-experiment-versions"
```
and re-used `$name` below that

just a thought/suggestion here, but i think it's a good suggestion :)